modules/trusted-firmware-m/Kconfig.tfm

```
@@ -310,6 +310,18 @@ config TFM_MCUBOOT_PATH_DOWNLOAD
 
 endchoice
 
+config TFM_ETHOS_DRIVER_PATH_LOCAL
+	string
+	prompt "Path to a locally available Ethos-U driver or an empty string"
```

Minor: could be combined into one line
```suggestion
	string "Path to a locally available Ethos-U driver or an empty string"
```